yarn.lock

```diff
@@ -6532,14 +6532,6 @@ bindings@^1.3.0, bindings@^1.5.0:
   dependencies:
     file-uri-to-path "1.0.0"
 
-"bip32-ed25519@git+https://github.com/Zondax/bip32-ed25519.git":
```

Comment:
If you don't install any new packages and you're just downloading modules, please `yarn install --frozen-lockfile`